src/wallet/wallet_rpc_server.cpp

```diff
@@ -138,12 +139,17 @@ namespace tools
         return true;
       if (boost::posix_time::microsec_clock::universal_time() < m_last_auto_refresh_time + boost::posix_time::seconds(m_auto_refresh_period))
         return true;
+      uint64_t blocks_fetched = 0;
       try {
-        if (m_wallet) m_wallet->refresh(m_wallet->is_trusted_daemon());
+        bool received_money = false;
+        if (m_wallet) m_wallet->refresh(m_wallet->is_trusted_daemon(), 0, blocks_fetched, received_money, true, 256);
```

Comment:
Could you make this `256` value a variable / macro to clarify its usage / make it easier to adjust please? Especially since this raw value is used later on in the code, for a new reader the connection may not be clear, and perhaps in the future one instance of the value will get changed and not the other, leading to some weird behavior.